src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -155,6 +158,13 @@ public async Task<IActionResult> SearchByUkprn(SearchByUkprnViewModel model)
                 {
                     validationMessage = UkprnValidationMessages.InvalidUkprn;
                 }
+                else
+                {
+                    if (!_ukprnWhitelistValidator.IsWhitelistedUkprn(ukprn))
```

Comment:
can you change to `else if` rather than a `if` buried within an `else`

Reply:
It's done